src/BuildQueue.cpp

```diff
@@ -903,6 +903,16 @@ namespace t2
       Log(kSpam, "Launching process");
       TimingScope timing_scope(&g_Stats.m_ExecCount, &g_Stats.m_ExecTimeCycles);
       ProfilerScope prof_scope(annotation, job_id);
+
+      const FrozenArray<FrozenFileAndHash>& outputFiles = node_data->m_OutputFiles;
+      uint64_t pre_timestamps[MAX_OUTPUT_NODES];
```

Comment:
you can use ThreadState.m_ScratchAlloc as a fast temp allocator, instead of limiting to MAX_OUTPUT_NODES

Reply:
Probably don't even need that - could use alloca() here?